assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -66,14 +66,15 @@ const ChatChannel = RestModel.extend({
     return !READONLY_STATUSES.includes(this.status);
   },
 
-  updateMembership(following, membership) {
+  updateMembership(membership) {
     this.current_user_membership.setProperties({
-      following,
+      following: membership.following,
       muted: membership.muted,
       desktop_notification_level: membership.desktop_notification_level,
       mobile_notification_level: membership.mobile_notification_level,
-      memberships_count: membership.user_count,
+      user_count: membership.user_count,
     });
+    this.set("memberships_count", membership.user_count);
```

Comment:
This is a weird case we have to handle, as ultimately channel is owner of this value but here we only get an updated membership and we have to reflect the new count to the parent channel

Reply:
No longer needed now, removed ;)

Reply:
🕺